run/events/.gitignore

```diff
@@ -0,0 +1,332 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do we need a .gitignore file specifically for this directory? It looks like we do have about 12 of them, but I don't know whether that's intended to be the way forward or not.
@amanda-tarafa what are your thoughts?

Reply:
I see, I didn't realize that. I can remove .gitignore then, it's not needed here.